Dockerfile

```diff
@@ -3,7 +3,7 @@ FROM golang:alpine as builder
 WORKDIR /build
 ADD . /build/
 
-RUN GOOS=linux GARCH=amd64 CGO_ENABLED=0 go build -mod=vendor -o api-server .
+RUN export GARCH="$(uname -m)" && if [[ ${GARCH} == "x86_64" ]]; then export GARCH="amd64"; fi && GOOS=linux GARCH=amd64 CGO_ENABLED=0 go build -mod=vendor -o api-server .
```

Comment:
I don't see how this make it build for other architectures 🤔 it still hardcode `amd64` doesn't it ? (also, `GARCH` is wrong, it should be `GOARCH` 🤔 )

Reply:
@vdemeester - Thanks for review.  I have updated code please review.